src/extractors/custom/www.asahi.com/index.js

```diff
@@ -0,0 +1,33 @@
+export const WwwAsahiComExtractor = {
+  domain: 'www.asahi.com',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
```

Comment:
Let's fetch the title as it displays on the page from its actual heading tag since it's easily accessible, often the `og:title` is slightly different.
In this case, `.ArticleTitle h1` should do.

Reply:
Thank you. Fixed.